nss-day-cohort-39/tabloid-full-stack-coulees
# comments

### Changes

- Created comment model, repo, and controller, edit form, delete form, and a model 

### To Test

- Click on the post details, you should see a "view comments" button
- If use has comments, then you should see them with them with 
** Subject
** Content
** Author's Display Name (automatically generated)
** Creation date (automatically generated)

- Then you should have an option to add a comment with the "Add Comment" button, you should be able to fill in 
** Subject
** Content
(Note: User should be able to cancel with "cancel" button and then be directed back to comment list)

- User should be able to edit and delete a comment
(Note: User should be able to cancel with "cancel" button and then be directed back to comment list)
